src/ClientUtilities/util/TestDomain.cs

```diff
@@ -33,6 +33,11 @@ public class TestDomain : ProxyTestRunner, TestRunner
 		/// </summary>
 		private DomainAgent agent;
 
+        /// <summary>
```

Comment:
Indentation in this file seems to be pretty evenly mixed between tabs and spaces -- my editor is set to use spaces and consequently this looks kind of ugly in this diff rendering. I didn't want to go in and convert the whole file one way or another (which would kind of obfuscate my changes here) but am happy to do so or switch my changes to using tabs if you care.